version3

KORALLLL opened the pull request.

NikiPshg committed:
feat: replace Whisper with Smart-Turn VAD, preserve diarization

NikiPshg committed:
feat(transcription): add new ASR models + ROVER integration

NikiPshg committed:
feat(speech-processing): major pipeline overhaul

- Integrated Vosk for speech recognition
- Optimized tone processing module
- Replaced NISQA-s to NISQA
- Removed deprecated audio clustering section

NikiPshg committed:
feat: optimizations and updates from pyannote 3.1 to pyannote/speaker-diarization-community-1

NikiPshg committed:
feat: removed support for launching via separate bash scripts

NikiPshg committed:
feat(libs): add NISQA and Smart-VAD libraries as local dependencies

NikiPshg committed:
feat: auto-download models on startup and add batching to music detector

NikiPshg committed:
feat(docs): minimum instructions for use

NikiPshg committed:
docs: minimum instructions for use

NikiPshg committed:
fix: correct LM download

NikiPshg committed:
fix(diarization): switching to speaker-diarization-community-1

NikiPshg committed:
fix(accent): fixed cuda support

KORALLLL commented on src/separation/musicdetector.py:
Зачем нужен файл model.py? В нём не то же самое?

NikiPshg replied:
Не заметил когда добавлял, удалил*

KORALLLL commented on src/transcription/transcripton_base.py:
```diff
@@ -0,0 +1,528 @@
+            repo_type='dataset',      
+            local_dir_use_symlinks=False  
+        )
+
```
выложи на мтусишную репу

KORALLLL replied:
и, если можно, модель фонемизации тоже

NikiPshg replied:
хорошо

KORALLLL commented on requirements_dev.txt:
есть варик перенести репы из строк 26-27 из твоих реп на мтусишные? Напиши мне, я форкну

KORALLLL reviewed (commented):
надо некоторые штуки решить, а так все хорошо

KORALLLL commented on requirements_dev.txt:
https://github.com/mtuciru/T-one-cuda-onnx
https://github.com/mtuciru/TryIPaG2P

поставь их пж

NikiPshg committed:
feat: addded ability to run collate without preprocess block

NikiPshg committed:
feat: replaced external links with mtuci-related ones

NikiPshg committed:
feat: update transcribation, separation for optimization

NikiPshg committed:
feat: removing Nisqa from libs, update transcription(tone)

NikiPshg committed:
feat: upgrade to gigaamv3

NikiPshg committed:
feat: dinamic consensus_num

NikiPshg committed:
feat: update imports, fix transcription block

NikiPshg committed:
feat: add volume normalization and deletion of files that do not go through the crest factor

NikiPshg committed:
docs: update readme

NikiPshg committed:
docs: update readme

NikiPshg committed:
feat(silence):add silence detect, update read me

NikiPshg committed:
feat: update order separation_yaml

Commit:
feat: change transription backend

Commit:
feat: fix bugs transcription

Commit:
feat: update nisqa work

Commit:
feat: update trt work and readme

NikiPshg committed:
feat: add distillMos and fix nisqa bug

SLENSER0 committed:
add benchmarking

NikiPshg committed:
feat: upload sortformer and remove pyannote,nisqa,silero

NikiPshg committed:
feat: update csv update in distillmos

NikiPshg committed:
feat: auto-load sortformer

NikiPshg committed:
feat: update config

NikiPshg committed:
fear: update transctipton trt cache path, clear repo

NikiPshg committed:
feat: update SmartVAD 3.0 --> 3.2

NikiPshg committed:
Merge pull request #9 from SLENSER0/benchmarking

add benchmarking

NikiPshg committed:
docs: update sortformer readme

NikiPshg committed:
feat: add webdataset support

NikiPshg committed:
feat: add trt for ruaccent

NikiPshg committed:
feat: add trt accent for each cuda

NikiPshg committed:
docs: update readme files

NikiPshg committed:
fix: fix benchmarking code

NikiPshg committed:
docs: update read me

KORALLLL commented on benchmarking/README.md:
желательно бы сгенерировать здесь понятно редми без слишком большого количества непонятных букв, чтобы можно быстро запустить

KORALLLL commented on docs/guide.md:
Это неактуально. Либо обновить, либо удалить. Лушче обновить, и объяснить, как использовать.

KORALLLL commented on docs/preparing.md:
тоже самое, что /docs/guide.md

KORALLLL commented on example/README.md:
а оно в веб-датасете работает сейчас?

KORALLLL commented on src/phonemizer/phonemizer.py:
Надо всем торчовым модельками сделать torch.compile

KORALLLL reviewed (commented):
пофиксить что я написал + заресолвить конфликты мержа

NikiPshg committed:
docs: update docs

NikiPshg committed:
feat: clean code, remove hardcode, add resume script, base.sh to caldi style

NikiPshg committed:
clean: remove old code

NikiPshg committed:
fix: fix download stage and update main readme

NikiPshg committed:
feat: update algorithm for cut audio, update csv now coll loudness_normalized

NikiPshg committed:
feat: optimization gpu util in preprocess

NikiPshg committed:
opt: speedup 13% smart turn with dlpack

NikiPshg committed:
feat: add csv manager

NikiPshg committed:
refactoring: remove duplicate code and update src readme

NikiPshg committed:
refactoring: refactortranscripton with dataset from torch

NikiPshg committed:
refactoring: refactor preprocess with dataset from torch

NikiPshg committed:
refactoring: refactor distilmos with dataset from torch

NikiPshg committed:
refactoring: clean code and add tmux for rover

NikiPshg committed:
docs: update dev readme

NikiPshg committed:
docs: update dev readme

Commit:
small changes

Commit:
docs: strict pipeline mode design spec

Commit:
docs: strict pipeline mode design spec

Commit:
checkpoint before vosk retry diagnostics

Commit:
config: add distillmos_filter subsection under separation

Commit:
feat(distillmos_filter): add Phase 1 — statistics and threshold determination

Commit:
fix(distillmos_filter): remove unused imports, add total_duration warnings, narrow return type, replace sys.exit with return None

Commit:
feat: integrate stage 5.5 (DistillMOS filter) into base.sh

- Replace integer comparison in stage_active() with bc-based float comparison
  to support floating-point stage numbers like 5.5
- Add stage 5.5 block between stages 5 and 6 that runs src.separation.distillmos_filter
- Update header comment with stage 5.5 entry

Commit:
small fixes

NikiPshg committed:
Merge pull request #10 from lab260ru/feature/bug_fixes

Feature/bug fixes

NikiPshg committed:
feat: add scripts analyze_balalaika_dataset

NikiPshg committed:
docs: update docs

NikiPshg committed:
feat: add denoising stage

NikiPshg committed:
chore: move to utils

NikiPshg committed:
feat: update CsvMerger

NikiPshg committed:
feat: add spectra-0 trt and fix preprocess memerror

NikiPshg committed:
feat: convert pandas to cuDF

KORALLLL reviewed (commented):
ok

NikiPshg committed:
feat: switch denoising inference to trt and update readme files

NikiPshg committed:
feat: add csv mode for find audio paths

NikiPshg committed:
add shard based processing, update readme files

NikiPshg committed:
import tqdm fix

NikiPshg committed:
chore: add more tqdm bars

NikiPshg committed:
update: update crest factor process 2.78x speed

NikiPshg committed:
feat: update rover processing and remove transcribations from balalaika.csv

NikiPshg committed:
speedup: num_workers for rover

NikiPshg committed:
fix(csv): serialize metadata writes and debug stage timings

NikiPshg committed:
feat: upload more data in parquet

NikiPshg committed:
fix: fix total duration cache

NikiPshg committed:
feat: add new asr metrics

NikiPshg committed:
feat: add support existing short autio process

NikiPshg committed:
update: split 5.6 in to 6 and 6.5 like DistillMOS inference and filter stage

NikiPshg committed:
feat: add inplace crest filter  and loudness for resolve io bottleneck

NikiPshg committed:
fix: fix hard replace result in balalaika.csv

NikiPshg committed:
Merge branch 'main' into dev
